docs/alerter_services.md

```diff
@@ -0,0 +1,17 @@
+# Alerter Services
+
+The idea about alerter services is to have a metrics driven approach of
+verifying the operators functionality. An operator might reconcile a system in
+which several resources should exist or maybe even not. As a means of
```

Comment:
>  An operator might reconcile a system in which several resources should exist or maybe even not.

I think for documentation like this, it's better to be a bit more explicit. What would you think of this instead:
`An operator reconciles a system were several resources exist.`

Reply:
Fine with me. Done.